### ISSUE TITLE
traini with my own data

### ISSUE THREAD

**Issue by johnlanbor** [eriklindernoren/PyTorch-YOLOv3#38]
Hi, i change the utils/datasets.py  and some config
i also make a mini train dataset which include 8 images and labels.
but when i train with it , i find the loss is very big(total loss 1000 +),so i cant get the right result for detection.
so, is there some possible bug in training code?

**Comment by johnlanbor** [comment]
i  update to the new code
the loss is start to converge but still cant get the correct result

**Comment by k123v** [comment]
I tried this code on my own dataset. But the mAp is much lower than SSD. I don't know where the problem is. Moreover, I run the code on VOC2007, the result is not good.

**Comment by eriklindernoren** [comment]
Hi, let's keep training issues in #2.
